plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -223,27 +224,21 @@ class ServiceActionDisabledModal extends React.Component {
     const command = this.getUpdateCommand();
 
     return (
-      <div>
-        <p>
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_1"
-          })}
+      <MountService.Mount
+        type={"ServiceEditMessage:Modal"}
+        getClipboardTrigger={this.getClipboardTrigger}
+        command={command}
```

Comment:
Let's rethink this. I don't think it makes sense to build the command here when it's not shown by default.